README.md

```diff
@@ -269,7 +327,14 @@ and you can provide interactive answers.
     edl -u
 ```
 
-*Note*: `edl` assumes `edi` is available on your PATH
+> [NOTE]
```

Comment:
If you write `> [!NOTE]` GitHub renders the block quote with a blue sidebar thingy and and a :information_source:-like icon. Adding the `!` shouldn't impact legibility for people reading the plain markdown file.

Also works with `> [!WARNING]` and `> [!IMPORTANT]`:

> [!NOTE]
> Highlights information that users should take into account, even when skimming.

> [!IMPORTANT]
> Crucial information necessary for users to succeed.

> [!WARNING]
> Critical content demanding immediate user attention due to potential risks.

See: https://github.com/orgs/community/discussions/16925